### ISSUE TITLE
How I can set up the reaction roles?

### ISSUE THREAD

**Issue by paolojpa** [Nookbot/Nookbot#30]
Hey! My question is How I can set up the reaction roles? Because I can’t see any command that complete that function.

**Comment by Phoenix1128** [comment]
You can find the answer to this in the comments of Issue https://github.com/Nookbot/Nookbot/issues/16.

**Comment by paolojpa** [comment]
> You can find the answer to this in the comments of Issue #16.

Okay if use this code

`.eval
client.reactionRoleDB.set('701203720200912966', {
  type: 'multiple',
  channel: '800065680685662258''
  reactions: {
    '%F0%9F%8E%AE': '588955307817041946',
    '%F0%9F%8E%8A': '588955576562614273',
  }
} `

How I can do that action?

**Comment by Phoenix1128** [comment]
> How I can undo that action? 

Undo? By removing the entry from the reaction roles database.

```js
.eval 
client.reactionRoleDB.delete('id');
```

**Comment by paolojpa** [comment]
Other thing, I’ve tried the code that you post 
`.eval client.reactionRoleDB.set('701203720200912966', {
  type: 'multiple',
  channel: '538940838441517068'
  reactions: {
    '%F0%9F%8E%AE': '588955307817041946',
    '%F0%9F%8E%8A': '588955576562614273',
  }
} `

But nothing happened (obviously I put my respective discord ids)

**Comment by Phoenix1128** [comment]
All that eval is doing is adding the message you wish to listen to reactions on to the database to be tracked. You have to react to the message you added, with the appropriate reaction, for it to function.

**Comment by paolojpa** [comment]
I have done that and it still doesn't react. ![image](https://user-images.githubusercontent.com/73677465/105121118-531cb180-5ab2-11eb-9659-d4c35ad5ddb0.jpeg)

**Comment by Phoenix1128** [comment]
Your reaction identifiers are incorrect. `'%F0%9F%8E%AE'` and `'%F0%9F%8E%8A'` are different emoji identifiers than what you want. Replace those with the identifiers for the reactions you want. In that example, the identifiers for `:confetti_ball:` and `:video_game:` are `'%F0%9F%8E%8A'` and `'%F0%9F%8E%AE'` respectively. You can use this eval for finding identifiers
```js
.eval 
return message.channel.messages.cache.get('messageID').reactions.cache.map((r) => r.emoji.identifier)
```
I've attached an example.
![Capture](https://user-images.githubusercontent.com/44852385/105123562-4a25e300-5aa6-11eb-93c3-be327524e251.PNG)

**Comment by paolojpa** [comment]
Okay. I will try with that.

**Comment by paolojpa** [comment]
Now I get this:![Cargando 105130891...]

**Comment by Phoenix1128** [comment]
That’s what you should get. Now the bot should add the role you defined when you click on its respective reaction.

**Comment by paolojpa** [comment]
Nothing happen :( 
Do you offer pay support?

**Comment by Phoenix1128** [comment]
No, I don't believe it's ethical to charge for support for a free to use, documented bot. Could you perhaps DM me on Discord so we can sort it out? My tag is Phoenix#0408.

**Comment by paolojpa** [comment]
Done! I've sent you a DM
